packages/@react-aria/i18n/src/useMessageFormatter.ts

```diff
@@ -12,7 +12,7 @@
 
 import {LocalizedStrings, MessageDictionary, MessageFormatter} from '@internationalized/message';
 import {useCallback, useMemo} from 'react';
-import {useLocale} from '../';
+import {useLocale} from '@react-aria/i18n';
```

Comment:
might be problematic, circular on itself. could present an issue in pnpm or yarn2

Reply:
I checked both pnpm and yarn 3, no issues presented themselves